api/service/model_endpoint_service.go

```diff
@@ -35,6 +35,9 @@ import (
 )
 
 const (
+	RealtimeDeploymentJob = "realtime"
```

Comment:
I think `model service deployment` is a better name

Reply:
ok I'll change this